### PR TITLE
bug fix: event api should parse events as json not yaml to honor expected data types

### PR DESCRIPTION
**Pull request by boriwo**

### CODE REVIEW

File: internal/pkg/tablemgr/routing_table_manager.go [hunk 1]
```diff
@@ -232,10 +232,12 @@ func (r *DefaultRoutingTableManager) RouteEvent(ctx context.Context, tid tenant.
 		event.WithTracePayloadOnNack(false),
 	)
 	if err != nil {
-		return errors.New("bad test event for route " + routeId)
+		return "", errors.New("bad test event for route " + routeId)
 	}
+	traceId, _, _ := e.GetPathValue("trace.id")
```
Comment (holidaymike) [inline internal/pkg/tablemgr/routing_table_manager.go:237]: Have u try?
```
traceId, ok := e.Context().Value(rtsemconv.EarsLogTraceIdKey).(string)
```

Comment (holidaymike) [reply internal/pkg/tablemgr/routing_table_manager.go:237]: Also, i wonder if we provide trace Id in the API call header? Does it propagate?

File: internal/pkg/app/handlers_v1.go [hunk 2]
```diff
@@ -224,14 +225,17 @@ func (a *APIManager) sendEventHandler(w http.ResponseWriter, r *http.Request) {
...
 		resp := ErrorResponse(convertToApiError(ctx, err))
 		resp.Respond(ctx, w, doYaml(r))
 		return
 	}
-	resp := ItemResponse(routeId)
+	item := make(map[string]string)
+	item["routeId"] = routeId
+	item["tx.traceId"] = traceId
+	resp := ItemResponse(item)
 	resp.Respond(ctx, w, doYaml(r))
```
Comment (holidaymike) [inline internal/pkg/app/handlers_v1.go:239]: Also, you *might* get trace.id for free if you respond with event's context (but then u will need to return the event instance from the RouteEvent method

Comment (boriwo) [reply internal/pkg/app/handlers_v1.go:239]: yeah i was wondering if that would always work - depending on timing the context may already be canceled, no?

**Review by holidaymike — APPROVED** [review]

**Pull request merged**